cmd/eval.go

```diff
@@ -296,6 +316,11 @@ func eval(args []string, params evalCommandParams, w io.Writer) (int, error) {
 		result.Profile = p.ReportTopNResults(params.profileLimit.v, sortOrder)
 	}
 
+	if params.coverage {
+		report := c.Report(parsedModules)
+		result.Coverage = &report
+	}
+
 	err = nil
```

Comment:
Huh? Ok. 😄

Reply:
Stray err = nil? Removed since the code below overwrites err anyways.